### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by egichuhis**
Hello, please review this PR.

Changes:

- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for your reusable components: /components.
- Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App>
- Add 2 routes in the application, each should render a page component:
/
/categories

Create components inside of the /components directory:

1. Contains individual book state, as well as a button to delete the book
2. Contains a list to renders individual books
3. Contains a form to create new books
4. Contains a navigation to link to the 2 routes you created

### CODE REVIEW

File: src/components/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import BooksList from './BooksList';
+import AddBook from './AddBook';
+
+const Home = () => (
+  <div className="container">
+    <Header />
+    <BooksList />
+    <AddBook />
+  </div>
+);
```
Comment (1GORDON) [inline src/components/Home.jsx:12]: Great job @egichuhis 🤝

- [ ] As stated in the project requirements, kindly add Navigation which links to the two routes. This should contain 2 `<Link>`s. One to `/` and one to `/categories`

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @egichuhis ✋

Good job so far!

## Highlights!
- react-router-dom is present in package.json ✔️ 
- There are 2 `<Route>`s added: `/` and `/categories` ✔️
- The App contains 2 page components: `<Books>` and `<Categories>` ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Home.jsx (hunk 1 above)
Comment (egichuhis) [reply src/components/Home.jsx:12]: This is already implemented and working, in the <Header> component and also in the app routing. Please see the screenshots below.

headerLinks

path

**Comment by egichuhis** [comment]
The requested change is already implemented and working, in the <Header> component and also in the BooApp routing. Please see the screenshots below.

headerLinks

path

File: src/components/BookStore.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <BrowserRouter>
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<Homepage />} />
+          <Route path="/categories" element={<Categories />} />
+        </Routes>
+      </div>
+    </BrowserRouter>
+  );
+}
```
Comment (roshan-bajgain) [inline src/components/BookStore.js:17]: - [ ] [Optional] One common pattern in React applications is to define and configure your application's routing in the "app.js" file. So please keep in mind from next milestone. Your "app.js" file is often considered the entry point of your application.
### Requirement
Screenshot 2023-09-23 093935

**Review by roshan-bajgain — APPROVED** [review]
Hi @egichuhis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good work ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**